controllers/kubeobjects.go

```diff
@@ -237,7 +270,8 @@ func restoreStatusProcess(
 	case velero.RestorePhasePartiallyFailed:
 		return errors.New("permanent: restore" + string(restore.Status.Phase))
 	case velero.RestorePhaseCompleted:
-		return w.restoreObjectsDelete(backupLocation, backup, restore)
+		// return w.restoreObjectsDelete(backupLocation, backup, restore)
+		return nil // keep objects around until backup is done (TODO: delete)
```

Comment:
`Restore` needs to be kept around for status with multiple sub-restores.  It's also ok to leave dummy `Backup` around because it'll get deleted the next time this backup number is used on this cluster.  I have been leaving the `BackupStorageLocation` deletions.  Leaving them around is ok if they don't change, but the logic to update them if they already exist isn't there.